document_page/readme.md

```diff
@@ -0,0 +1,34 @@
+Module's migration
```

Comment:
This should be in RST format with name README.rst and follow this template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst